src/lib/components/submit/SubmitSelect.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+    import ExternalLink from '$lib/components/ui/ExternalLink.svelte';
+    import ProposalSelector from '$lib/components/proposals/ProposalSelector.svelte';
+	import SubmitContinue from '$lib/components/submit/SubmitContinue.svelte';
+	import { createEventDispatcher } from 'svelte';
+
+    export let proposalType: string | undefined;
+
+    const dispatch = createEventDispatcher();
+    const select = () => dispatch('pnwrkSelect', { proposalType });
+</script>
+
+<h1 class="mb-12 text-6xl font-bold capitalize">Select Proposal Type</h1>
+
+<style>
+    .flex-container {
+      display: flex;
+      gap: 1rem; 
+    }
+</style>
+  
+<div class="flex-container">
+    <div class="mb-4">
```

Comment:
why do you need those `div`? if it styling purpose, seems that it can be resolved with a row gap